purchase_security_advanced/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "13.0.2.0.0",
+    "version": "13.0.2.0.1",
```

Comment:
This is a new feature (restricting access to contacts across the application depending on groups belonging), not a bugfix IMO.